Source/WebCore/style/AnchorPositionEvaluator.cpp

```diff
@@ -498,6 +498,11 @@ LayoutRect AnchorPositionEvaluator::computeAnchorRectRelativeToContainingBlock(C
             anchorLocation.moveBy(-anchorBox->view().frameView().scrollPositionRespectingCustomFixedPosition());
     }
 
+    if (CheckedPtr containingBox = dynamicDowncast<RenderBox>(containingBlock)) {
```

Comment:
I believe this doesn't need to be a CheckedPtr, because the containingBlock is a reference so we can rely on it existing already. So it can just be `auto`. (I _think_ that's how this works, maybe check with someone else...)

Reply:
`CheckedPtr` doesn't participate in reference counting (i.e it doesn't extend the lifetime of the containing block), it just makes sure the containing block lives for the entire duration of the `CheckedPtr`. So it's perfectly fine to use it here.